RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
+public enum SequenceMaterializeResult<T> {
```

Comment:
The rename totally make sense, I'll do that. In terms of `errors` do you mean in the enum case name or in the associated value parameter?

I'm not sure plural `errors` makes sense given it's only a single error that terminates the sequence? (I think, Rx newbie here).

1. In case name:

```
public enum MaterializedSequenceResult<T> {
    case completed(elements: [T])
    case errors(elements: [T], error: Error)
}
```

2. In associated value name

```
public enum MaterializedSequenceResult<T> {
    case completed(elements: [T])
    case failed(elements: [T], errors: [Error])
}
```